logconfig/logconfig.go

```diff
@@ -0,0 +1,30 @@
+package logconfig
+
+import (
+	"context"
+
+	"github.com/anz-bank/pkg/log"
+)
+
+type isVerboseLoggingKey struct{}
+type isVerboseLogging struct {
+	Flag bool
+}
+
+// Get whether or not sysl-go internals log additional debug/verbose level information.
+func IsVerboseLogging(ctx context.Context) bool {
+	v, ok := ctx.Value(isVerboseLoggingKey{}).(*isVerboseLogging)
+	if ok {
+		return v.Flag
+	}
+	return false
+}
+
+// Set against the context whether or not sysl-go internals log additional debug/verbose level information.
+func SetVerboseLogging(ctx context.Context, verbose bool) context.Context {
+	ctx = log.WithConfigs(log.SetVerboseMode(verbose)).Onto(ctx)
```

Comment:
set the verbose mode to context as well, otherwise, the `field.Debug` `requestlogger.FlushLog()` cannot be logged.

Reply:
I don't know if this is required. The `pkg` logger stored against the old context should already be `verbose`. If the state isn't being transferred correctly then you might want to dig into the interaction between `log.From` and `log.WithLogger`.